integreat_compass/settings.py

```diff
@@ -0,0 +1,115 @@
+"""
+Django settings for integreat_compass project.
+
+Generated by 'django-admin startproject' using Django 4.1.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-cy1_$eh*&b&@t9^13^*#el48a(^n^=(%s%shbpzyxe3y49qh+("
```

Comment:
```suggestion
SECRET_KEY = os.environ.get("INTEGREAT_CMS_SECRET_KEY", "dummy" if DEBUG else "")
```
I like the security measure we have for integreat-cms here... this prevents us from accidentally deploying this with the insecure dummy key, because an empty key will immediately raise an error if the correct key is not passed via the environment.